jenkins-scripts/ign_gazebo-default-devel-windows-amd64.bat

```diff
@@ -5,7 +5,7 @@ set VCS_DIRECTORY=ign-gazebo
 set PLATFORM_TO_BUILD=x86_amd64
 set IGN_CLEAN_WORKSPACE=true
 :: dlfcn
-set DEPEN_PKGS="dlfcn-win32 cuda curl jsoncpp freeimage ogre gts glib fcl eigen3 ccd assimp libyaml libzip gflags protobuf tinyxml2"
+set DEPEN_PKGS="dlfcn-win32 cuda cppzmq curl jsoncpp ffmpeg freeimage ogre qt5 qwt gts glib fcl eigen3 ccd assimp libyaml libzip gflags protobuf tinyxml2 zeromq"

```

Comment:
Do all transient dependencies need to be listed here? Wouldn't other Ignition libraries bring these?

Reply:
We don't currently have a way of importing vcpkg dependencies from software dependencies nicely. So yes, we would need to list here all transition dependencies here by now or implement a way of doing it in these scripts.